dev/io.openliberty.microprofile.metrics.internal.3.0_fat/build.gradle

```diff
@@ -0,0 +1,95 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
```

Comment:
Don't think the contents of this file needed to be copied over. This was testing something else.